stable/cockroachdb/values.yaml

```diff
@@ -72,3 +72,5 @@ Locality: ""
 ExtraArgs: []
 # ExtraSecretMounts is a list of names from secrets in the same namespace as the cockroachdb cluster, which shall be mounted into /etc/cockroach/secrets/ for every cluster member.
 ExtraSecretMounts: []
+# ExtraEnvArgs is a list of name,value tuples providing extra ENV variables
```

Comment:
It's not clear to me from this comment how I should format the environment variables into an array, given that each has a name and a value. Mind clarifying it with an example or something?

Reply:
done